client/spa/js/instructor/instructor.html

```diff
@@ -0,0 +1,21 @@
+<style type="text/css">
+.main {
+  margin: auto;
+  max-width: 980px;
+}
+#result {
+  padding: 1em 0;
+}
+.success {
+  color: green;
+}
+</style>
+<section id="instructor">
+  <div class="main container">
+    <h1><%- firstName %> <%- lastName %></h1>
+    <p><%- skills %></p>
+    <button class="edit">Edit</button>
```

Comment:
When I view an instructor id, I don't see the edit button anymore. Must be a conflict with our class names when my branch was merged with master. I have the same class name of edit, and you have <button class="edit">Edit</button> in your template. We can either prefix our class names like i-edit for yours and lr-edit for mine. Or you can use my template if you like to have the same class names.